tools/nightly.py

```diff
@@ -486,10 +507,10 @@ def move_nightly_files(spdir, platform):
     target_dir = os.path.abspath("torch")
     listing = _get_listing(source_dir, target_dir, platform)
     # copy / link files
-    if platform.startswith("win"):
-        _copy_files(listing, source_dir, target_dir)
-    else:
+    if _can_link(platform, source_dir, target_dir):
         _link_files(listing, source_dir, target_dir)
+    else:
+        _copy_files(listing, source_dir, target_dir)
```

Comment:
Wouldn't `try: except:` paradigm be better than `if: then:` in this case?
```suggestion
    try::
        _link_files(listing, source_dir, target_dir)
    except OSError:
        _copy_files(listing, source_dir, target_dir)
```

Reply:
The try is in the `_can_link()` function. There is nothing particularly exceptional about choosing which function should be used to transfer files.

Reply:
This approach eliminate the need of `_can_link` function completely, isn't it?